packages/renderer/src/lib/image/ImagesList.svelte

```diff
@@ -133,9 +133,10 @@ async function deleteSelectedImages() {
 let refreshTimeouts: NodeJS.Timeout[] = [];
 const SECOND = 1000;
 function refreshAge() {
-  images = images.map(imageInfo => {
-    return { ...imageInfo, age: imageUtils.refreshAge(imageInfo) };
+  images.forEach(imageInfo => {
```

Comment:
might be better to use for of rather than forEach in case at some point we need async stuff, etc

Reply:
(and it's faster)

Reply:
Switched to for loop.